README.md

```diff
@@ -14,6 +14,17 @@ through a so called _version-file_. The process of crafting such a version file,
 updated, is a bit cumbersome. 
 This package aims to make the creation of such a version file easier.
 
+The key/value pairs that be specified in the version file and [their official meaning](https://docs.microsoft.com/en-us/windows/win32/menurc/stringfileinfo-block) are shown the following table:
+
+| Parameter Name | Description |
+|:---:|---|
+| CompanyName | Name of the company that produced the file, for example, "My Imaginary Company, Inc.". |
+| FileDescriptionFile | Description to be presented to users. It may be displayed when the user is choosing files to install. For example, "A simple app that does simple things.". |
```

Comment:
```suggestion
| FileDescription | Description to be presented to users. It may be displayed when the user is choosing files to install. For example, "A simple app that does simple things.". |
```